auth/auth_test.go

```diff
@@ -880,6 +884,20 @@ func TestVerifyIDTokenAndCheckRevokedError(t *testing.T) {
 	}
 }
 
+func TestVerifyIDTokenAndCheckDisabledError(t *testing.T) {
+	s := echoServer(testGetDisabledUserResponse, t)
+	defer s.Close()
+	revokedToken := getIDToken(mockIDTokenPayload{"uid": "uid", "iat": 1970})
+	s.Client.idTokenVerifier = testIDTokenVerifier
+
+	p, err := s.Client.VerifyIDTokenAndCheckRevoked(context.Background(), revokedToken)
```

Comment:
Just wondering. Do we have unit test coverage for `VerifySessionCookie()` with this change? We might want to add that too.

Reply:
The requested change was only to verifyIDToken, but regardless the two should function identically in unit tests (practically speaking, the existing unit test that was added is actually verifying that checkRevokedOrDisabled works properly.  verifyIDToken and VerifySessionCookie just directly return the error that they get from checkRevokedOrDisabled).

Reply:
True. But `VerifySessionCookie()` is a separate public API with its own tests. See https://github.com/firebase/firebase-admin-go/blob/928b104e521ffc3d7de0319efe1553c31e85e13f/auth/auth_test.go#L1074

This PR is changing its behavior (rightly so), but without the proper test coverage we won't be able to detect future breakages. Imagine a scenario where we refactor how the `checkRevokedOrDisabled()` is used internally.  May be we decide to not call it from `VerifySessionCookie()`. The resulting behavior change will go unverified without a test case.

Also, for reference here's the equivalent PR in Java that we did: https://github.com/firebase/firebase-admin-java/pull/585/files. You will notice how there are separate tests for `VerifyIdToken()` and `VerifySessionCookie()`.

Reply:
Hey @Galadros, @hiranya911 is right. I also recommend you add another test for `VerifySessionCookie()` for the exact same reason.

Reply:
Done.